tests/Feature/TaskControllerTest.php

```
@@ -40,7 +40,7 @@ public function tasks_index_shows_pending_tasks()
         $user = User::factory()->admin()->create();
         $matter = Matter::factory()->create(['dead' => false]);
         $event = Event::factory()->filing()->forMatter($matter)->create();
-        $task = Task::factory()->pending()->forEvent($event)->create(['code' => 'DL']);
+        $task = Task::factory()->pending()->forEvent($event)->create(['code' => 'REP']);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$task'. (undefined)

(UnusedLocalVariable)